src/components/text-editor/__internal__/toolbar/toolbar.spec.js

```diff
@@ -60,8 +60,8 @@ describe("Toolbar", () => {
           userSelect: "none",
           order: "2",
           border: "none",
-          backgroundColor: baseTheme.editor.toolbar.background,
```

Comment:
I guess you could also remove `baseTheme` and `ThemeProvider` from this spec file